recipe/meta.yaml

```diff
@@ -23,6 +23,8 @@ requirements:
     - setuptools >=45
     - setuptools-scm >=4
```

Comment:
It was here, though 🤔

Reply:
dunno.
if you inspect the resulting tarballs (download one from https://anaconda.org/conda-forge/mpi4jax ) you can see that metada has version 0.0.0